react/features/conference/functions.js

```diff
@@ -0,0 +1,29 @@
+import { getName } from '../app';
+import { translateToHTML } from '../base/i18n';
+import { RTCBrowserType } from '../base/lib-jitsi-meet';
+import { showWarningNotification } from '../notifications';
+
+/**
+ * Shows the suboptimal experience notification if needed.
+ *
+ * @param {Function} dispatch - The dispatch method.
+ * @param {Function} t - The translation function.
+ * @returns {void}
+ */
+export function maybeShowSuboptimalExperienceNotification(dispatch, t) {
+    if (!RTCBrowserType.isChrome() && !RTCBrowserType.isFirefox()) {
```

Comment:
Maybe you have to explicitly check electron too? It might be worthwhile to move just this check into a helper function for clarify...or maybe not.

Reply:
Oh. Good catch! Maybe NWJS too. I'm going to think about it and review the other browser types that we detect.